src/pages/home/report/ReportActionsView.js

```diff
@@ -41,6 +42,7 @@ const propTypes = {
     ...windowDimensionsPropTypes,
     ...withDrawerPropTypes,
     ...withLocalizePropTypes,
+    ...policyPropTypes,
```

Comment:
I don't think we can use `policyPropTypes` because it has `policyMemberList` which we don't pass now.

Can we use the way you previously had? @hungvu193